exercises/practice/zebra-puzzle/.docs/instructions.append.md

```diff
@@ -0,0 +1,13 @@
+# Implementation
+
+Implement the `waterDrinker` and `zebraOwner` methods from the `ZebraPuzzle`
+class. They must return a string each, whose values are the answers to the
+zebra-puzzle questions "Who drinks water?" and "Who owns the Zebra?".
+Each answer will be one of the resident's nationalities:
+Englishman, Spaniard, Ukrainian, Norwegian, or Japanese.
+
+Obviously, you could simply write two single-statement function
+if you peek at the test program to see the expected solution.
+But the goal is to develop an algorithm which uses
+the given facts and constraints for the puzzle
+and determines the two correct answers.
```

Comment:
Exercisms Markdown rules say to do "one sentence per line". We try to apply that for all the new content that we add. Would be great if you could do this here as well.

Reference: https://exercism.org/docs/building/markdown/markdown#h-layout